xml/System.ServiceModel/WSDualHttpBinding.xml

```diff
@@ -711,7 +711,7 @@
    
   
 ## Examples  
- The following examples show how to get the <xref:System.ServiceModel.WSDualHttpBinding.System.ServiceModel.Channels.IBindingRuntimePreferences.ReceiveSynchronously%2A> value.  
+ The following examples show how to get the <xref:System.ServiceModel.WSDualHttpBinding.System%23ServiceModel%23Channels%23IBindingRuntimePreferences%23ReceiveSynchronously%2A> value.  
```

Comment:
`MaxReceivedMessageSize` is inherited from `HttpBindingBase`. We had to fix a couple of these back in *Migration Fun Week*. It could go something like this (but IIRC [@]rpetrusha said this might be too shocking to throw the reader like this) ...

> Note that the settings for \<xref:System.ServiceModel.WSDualHttpBinding.MaxBufferPoolSize%2A> and \[\`BasicHttpBinding.MaxReceivedMessageSize\`](xref:System.ServiceModel.HttpBindingBase.MaxReceivedMessageSize%2A) are local behavioral settings.

... OR call it out more explicitly (which isn't quite so shocking) ...

> Note that the settings for \<xref:System.ServiceModel.WSDualHttpBinding.MaxBufferPoolSize%2A> and \`BasicHttpBinding.MaxReceivedMessageSize\` (inherited from \[\`HttpBindingBase.MaxReceivedMessageSize\`](xref:System.ServiceModel.HttpBindingBase.MaxReceivedMessageSize%2A)) are local behavioral settings.

... OR just directly to `HttpBindingBase` (but that disconnects the passage from `BasicHttpBinding` perhaps too much; i.e., the reader won't be using the `HttpBindingBase` directly) ...

> Note that the settings for \<xref:System.ServiceModel.WSDualHttpBinding.MaxBufferPoolSize%2A> and \<xref:System.ServiceModel.HttpBindingBase.MaxReceivedMessageSize%2A> are local behavioral settings.